packages/backend-defaults/src/entrypoints/httpRouter/httpRouterServiceFactory.ts

```diff
@@ -63,9 +73,15 @@ export const httpRouterServiceFactory = createServiceFactory({
     router.use(credentialsBarrier.middleware);
     router.use(createCookieAuthRefreshMiddleware({ auth, httpAuth }));
 
+    const pluginRoutes = PromiseRouter();
+    router.use(pluginRoutes);
+
+    const middleware = MiddlewareFactory.create({ config, logger });
```

Comment:
Perhaps for later, but I think we should probably move the error handler out of the `MiddlewareFactory`, given that it's used in a couple of different places. This one is really only intended for the root HTTP service.

Reply:
Agreed, it's a bit sneaky to take it out of where it's clearly exported by the root router. We should keep it in mind at the very least.